modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -34,36 +34,73 @@ Establishing the right performance expectations refers to the fact that the real
 == Provisioning a worker with real-time capabilities
 
 . Install Performance Addon Operator to the cluster.
+
 . Optional: Add a node to the {product-title} cluster.
 See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/8/html-single/tuning_guide/index#Setting_BIOS_parameters[Setting BIOS parameters].
-. Optional: Create a new machine config pool for real-time nodes.
-. Add the node to the proper machine config pool, using node role labels.
-+
-You must decide which nodes will be configured with real-time workloads. It could be all of the nodes in the cluster or a subset of the nodes. The Performance Addon Operator expects all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, you just point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
 
-. Create the `PerformanceProfile` with the proper set of housekeeping cores and `realTimeKernel: enabled: true`.
+. Add a label `worker-rt` to the worker nodes that require the real-time capability using the `oc` command.
 
-. Specify a node selector in the `PerformanceProfile`, as shown here:
+. Create a new machine config pool for real-time nodes:
 +
 [source,yaml]
 ----
-apiVersion: performance.openshift.io/v2
-kind: PerformanceProfile
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
 metadata:
- name: example-performanceprofile
+  name: worker-rt
+  labels:
+    machineconfiguration.openshift.io/role: worker-rt
 spec:
-...
-  realTimeKernel:
-    enabled: true
+  machineConfigSelector:
+    matchExpressions:
+      - {
+           key: machineconfiguration.openshift.io/role,
+           operator: In,
+           values: [worker, worker-rt],
+        }
+  paused: false
   nodeSelector:
-    node-role.kubernetes.io/worker-rt: ""
+    matchLabels:
+      node-role.kubernetes.io/worker-rt: ""
 ----
+Note that a machine config pool worker-rt is created for group of nodes that have the label `worker-rt`.
+
+. Add the node to the proper machine config pool, using node role labels.
++
+You must decide which nodes will be configured with real-time workloads. It could be all of the nodes in the cluster or a subset of the nodes. The Performance Addon Operator expects all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, you just point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
```

Comment:
```suggestion
You must decide which nodes are configured with real-time workloads. You could configure all of the nodes in the cluster, or a subset of the nodes. The Performance Addon Operator expects that all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, you just point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
```

Reply:
Should this be a note?

Reply:
@maxwelldb this content was already there as part of step 5, but yes imo I can make it a note

Reply:
Your call! If it's not in scope, maybe put a pin in it.

Reply:
I will keep it as a note , as this looks like useful info for the user.